### PR TITLE
pythonPackages.fastpbkdf2: init at 0.2

### PR DESCRIPTION
**Pull request by jqueiroz**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by FRidh** [comment]
@GrahamcOfBorg build pythonPackages.fastpbkdf2

**Comment by FRidh** [comment]
@GrahamcOfBorg build python3Packages.fastpbkdf2

File: pkgs/development/python-modules/fastpbkdf2/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchFromGitHub, buildPythonPackage
+, openssl, pytest, cffi, six }:
+
+buildPythonPackage rec {
+  pname = "fastpbkdf2";
+  version = "0.2";
+  src = fetchFromGitHub {
```
Comment (dotlambda) [inline pkgs/development/python-modules/fastpbkdf2/default.nix]: Why not `fetchPypi`?

**Review by dotlambda — CHANGES_REQUESTED** [review]
Please add the usual blank lines to the expression.

File: pkgs/development/python-modules/fastpbkdf2/default.nix (hunk 1 above)
Comment (jqueiroz) [reply pkgs/development/python-modules/fastpbkdf2/default.nix]: The package has tests on Github, but not on PyPI. In cases such as this, is it still better to switch to PyPI?

Comment (dotlambda) [reply pkgs/development/python-modules/fastpbkdf2/default.nix]: In that case, it's okay to use GitHub. However, please add a comment with the reason.

File: pkgs/development/python-modules/fastpbkdf2/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    owner  = "Ayrx";
+    repo   = "python-fastpbkdf2";
+    rev    = "5c4ff9064c78770519db7e28c61a08eb926fecec";
```
Comment (dotlambda) [inline pkgs/development/python-modules/fastpbkdf2/default.nix]:
```suggestion
    rev    = "v${version}";
```

**Review by dotlambda — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/fastpbkdf2/default.nix (hunk 2 above)
Comment (jqueiroz) [reply pkgs/development/python-modules/fastpbkdf2/default.nix]: Changed, thanks for the suggestion.

File: pkgs/development/python-modules/fastpbkdf2/default.nix (hunk 1 above)
Comment (jqueiroz) [reply pkgs/development/python-modules/fastpbkdf2/default.nix]: Okay, I added a comment explaining the choice.

**Comment by dotlambda** [comment]
@GrahamcOfBorg build python2.pkgs.fastpbkdf2 python3.pkgs.fastpbkdf2

**Pull request merged**